src/ReflectionMethod.php

```diff
@@ -224,6 +250,10 @@ public function invokeArgs($object, array $args)
      */
     public function isAbstract()
     {
+        if (!$this->functionLikeNode) {
```

Comment:
When this is possible?

Same comment applies to other methods in the class.

Reply:
This is when the PHP source code doesn't exist, so we defer to the output of the native implementation.